### DISCUSSION TITLE
GPU-hosted Runners - Public Beta Feedback 🚀

### DISCUSSION THREAD

**Discussion by grandmas-favorite** [community/community#128041] [Actions] [answered]
Hi Community!

We’re delighted to unveil [GPU hosted runners](https://github.blog/2024-04-02-bringing-enterprise-level-security-and-even-more-power-to-github-hosted-runners/#gpu-hosted-runners-available-in-public-beta-%f0%9f%8e%89) is now in public beta across GitHub Team and Enterprise Cloud plans! 

⚙️ This new runner empowers teams working with machine learning models, such as large language models (LLMs) or those requiring GPU graphic cards for game development, to run these more efficiently as part of their automation or CI/CD process. This allows teams to do complete application testing, including the ML components, with GitHub Actions.

### Documentation and Resources
To help you get started, we have prepared comprehensive documentation and resources:

*   🗞️    **GitHub Blog**: Read about the full release on the GitHub change log, [here](https://github.blog/2024-04-02-bringing-enterprise-level-security-and-even-more-power-to-github-hosted-runners/#gpu-hosted-runners-available-in-public-beta-%f0%9f%8e%89).

*   📊  **Docs and Monitoring**: Find out more information on how to set up GPU our public [documentation](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions). 

*   📝   **More Feedback**: To share your feedback and help us find the right additional GPU SKUs to support, please fill out this [form](https://forms.gle/9JQ3rtm1pX6RcEjt8)

### Participate in the Public Beta!
Your feedback during this beta phase is invaluable to us. We encourage you to share your experiences here, which will be instrumental in refining and enhancing this feature as we look toward the future.

Stay tuned for updates and enhancements throughout the beta period!

**Comment by IhfazAlam** [comment]
Hello

**Reply by IhfazAlam** [reply]
Follow me

**Comment by samus-aran** [comment, marked as answer]
## GPU-hosted Runners Now Generally Available!

Check out our [Changelog blog post](https://github.blog/changelog/2024-07-08-github-actions-gpu-hosted-runners-are-now-generally-available/) for more details 👈
